src/views/create/index.js

```diff
@@ -50,29 +114,44 @@ class Create extends Component {
                         <form className="pass-form">
                             <div className="cont-input">
                                 <Input type="password" name="passphrase" id="passp" placeholder="•••••••••••••••••" />
-                                <span id="passphraseErrorLabel" className="error passphrase-error"> <img src={altertR} alt="alert" />The passphrase does not fit the requirements</span>
+                                <span id="error-label" className="error passphrase-error"> <img src={altertR} alt="alert" /></span>
                             </div>
                             <div className="btn-subm">
-                                <Button onClick={this.handleCreateAccount} >Download</Button>
-                                <Button onClick={this.handleCreateAccount} dark>Create</Button>
+                                <Button onClick={this.handleCreateAccount} >Create</Button>
                             </div>
                         </form>
-                        <a href="/" className="account">Already have an account? Access my Pocket account</a>
+                        <div style={{display: "inline", maxWidth: "500px"}} className="cont-input">
+                            <label style={{fontWeight: "bold"}} htmlFor="prk">PRIVATE KEY</label>
+                            <Button id="download-key-file" className="download-btn" onClick={this.handleDownload} >Download Key File</Button>
+                        </div>
+                        <a style={{display: "block", marginTop: "26px"}} 
+                        href="/import" className="account">Already have an account? Access my Pocket account</a>
                     </div>
                     <div className="alert">
                         <img src={altertT} alt="alert" />
                         <div className="cont-alert">
                             <div className="title">
-                                <h3>Don’t forget to save your PASSPHRASE!</h3>
+                                <h3>STORE SAVE YOUR PRIVATE KEY!</h3>
                             </div>
                             <p>
-                                You wont be able to change it, make a back up, store it save and preferably offline. You’ll nee it to import or set up your node.
-                                <br />
-                                <br />
-                                Remember that  the keyfile by itself is useless without the passphrase.
+                            You wont be able see it again or change it, make a back up, store it save  preferably offline. You’ll need it to acces your account again.
```

Comment:
You won't be able see it again or change it -- make a back up and store it safely, preferably offline. You’ll need it to access your account again.